### ISSUE TITLE
[KEYBOARD]: Links in the chat log must be keyboard focusable

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#25156]
# [508-defect-1](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a person who prefers to navigate by keyboard, I need the links in the chatbot log to receive focus when I press the TAB key.

## Details

The links added to the VA chatbot's responses are not reachable by keyboard when I am navigating through the page with the TAB key. Quicktime movie attached below.

## Acceptance Criteria

- [ ] Links can be confirmed to receive keyboard focus when navigating with the TAB key
- [ ] This can be verified by typing `document.activeElement` into the Chrome DevTools console

## Environment

* https://staging.va.gov/virtual-agent/

## WCAG or Vendor Guidance (optional)

* https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-keyboard-operable.html

## Screenshots or Trace Logs

https://user-images.githubusercontent.com/934879/119392255-24e3f400-bc95-11eb-96ee-50da95041cde.mov

**Comment by anitadewitt-va** [comment]
We cannot make links in the chat log tabbable. We tried adding tab indexes to the links, but this does not solve this issue. This would need to be addressed by the Microsoft webchat team.

**Comment by 1Copenut** [comment]
@anitadewitt-va This would be worth escalating with the MSFT chatbot team, probably as an issue ticket that you can point back to this one if that's appropriate.

**Comment by ianpsant** [comment]
Filed Microsoft ticket mentioned above.

**Comment by shiragoodman** [comment]
hey @ianpsant - it's been a while. Any update from Microsoft on this or other issues?

**Comment by ianpsant** [comment]
Microsoft addressed this issue with their PR (https://github.com/microsoft/BotFramework-WebChat/pull/4108) which added an instruction page showing the user how to correctly navigate so that the chat log could be keyboard focusable. We implemented this on our chatbot on ticket https://github.com/department-of-veterans-affairs/va-virtual-agent/issues/390. We will revisit this more when we move the chatbot and this goes through another accessibility review.
